metrics/wer/wer.py

```diff
@@ -91,4 +91,10 @@ def _info(self):
         )
 
     def _compute(self, predictions, references):
-        return wer(references, predictions)
+        incorrect = 0
+        total = 0
+        for prediction, reference in zip(predictions, references):
+            measures = compute_measures(reference, prediction)
+            incorrect += measures["substitutions"] + measures["deletions"] + measures["insertions"]
+            total += measures["substitutions"] + measures["deletions"] + measures["hits"]
+        return incorrect / total
```

Comment:
Just for safety, you may want to handle the case of `total = 0` (e.g., empty set). Unit tests to verify the change would be great as well. Thanks for making this change!

Reply:
Thanks @elgeish for your comments.

The reasons why I have not explicitly handled the edge case `total = 0` are:
- `compute_measure` does already raise a ValueError exception if any of the `references` is the empty string: see https://github.com/jitsi/jiwer/blob/f8e5404e4ddb7259081191443def1b6670480244/jiwer/measures.py#L242
- if `references` and `predictions` have length = 0, the previous behavior was also raising the same ValueError; the current implementation will raise a ZeroDivisionError in this case

In relation to unit tests, currently we do not test scripts. Maybe @lhoestq can give more insight on this.

Reply:
Sounds good; makes sense!